desktop/package/linux/package.sh

```diff
@@ -117,7 +119,7 @@ fi
 echo Generating rpm package
 $JAVA_HOME/bin/javapackager \
     -deploy \
-    -BappVersion=$version_base \
```

Comment:
@ripcurlx I seem to recall that I used `$version_base` which stripped the "-SNAPSHOT" portion because it causes trouble when generating the package. Maybe I forgot to commit where it sets `$version_base`...

Reply:
Feel free to add this anytime again 😄 I just changed what I needed to be able to create the builds.